pandokia/helpers/importer.py

```diff
@@ -1,8 +1,20 @@
 '''import an arbitrary file with an arbitrary module name
 '''
 import sys
-import imp
 
+import importlib.util
+import importlib.machinery
+
+# replacement for imp.load_source() from https://docs.python.org/dev/whatsnew/3.12.html#imp
```

Comment:
can you note in a comment here ALL the places where this func is written separately please?  Pretty sure I just saw this get added in a stewey or other PR.

Reply:
NOT asking that they be consolidated.  Just comment-connected

Reply:
I've removed the repetition by moving the function to common.py and importing it elsewhere

Reply:
Thanks!  Can the one in the other repo (stewey?) get a comment pointing it here? (and here there)   Not asking for that one to change to an import.

Reply:
I've put linking documentation in both repos.